src/xwayland/xserver.rs

```diff
@@ -290,72 +300,79 @@ fn xwayland_ready<WM: XWindowManager>(inner: &Rc<RefCell<Inner<WM>>>) {
     }
 }
 
-pub(crate) enum Void {}
-
 /// Exec XWayland with given sockets on given display
 ///
 /// If this returns, that means that something failed
```

Comment:
This doc comment needs to be updated as well

Reply:
done